app/models/miq_provision_request_template.rb

```diff
@@ -74,6 +74,7 @@ def get_parent_task(service_task)
   end
 
   def number_of_vms_from_dialog(root_svc, parent_task)
+    return nil unless root_svc.options[:dialog]
```

Comment:
@Fryguy 
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_provision_request_template.rb#L59
we should be ok, since we check if the value is blank before converting it to integer